processing/src/test/java/org/apache/druid/math/expr/ConstantExprTest.java

```diff
@@ -19,20 +19,139 @@
 
 package org.apache.druid.math.expr;
 
+import org.apache.druid.segment.column.TypeStrategies;
+import org.apache.druid.segment.column.TypeStrategiesTest.NullableLongPair;
+import org.apache.druid.segment.column.TypeStrategiesTest.NullableLongPairTypeStrategy;
 import org.apache.druid.testing.InitializedNullHandlingTest;
 import org.junit.Test;
 
+import java.math.BigInteger;
+
+import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotSame;
 import static org.junit.Assert.assertSame;
 
 public class ConstantExprTest extends InitializedNullHandlingTest
 {
   @Test
-  public void testLongSingleThreadedExpr()
+  public void testArrayExpr()
+  {
+    checkExpr(
```

Comment:
How about an array of string? Now I'm wondering if the strings are properly escaped.

Reply:
for `toString` its not escaped; however `stringify` looks good!